server/channels/app/worktemplates/templates.yaml

```diff
@@ -42,7 +42,7 @@ content:
       template: "Product Release"
       name: "Feature release"
       id: product-release-playbook
-      illustration: "/static/worktemplates/product_teams/feature_release/playbook.png"
```

Comment:
Oh sorry, the typo was that it's missing an `s` at the end. The new file I had added is: https://github.com/mattermost/mattermost-server/blob/master/webapp/channels/src/images/worktemplates/product_teams/feature_release/playbooks.png

Reply:
There was an existing one (same picture, same size) in the `playbooks/` folder. That's a pattern all other templates respect so I made sure to stick to it

Reply:
Really? I can't see the diff lol

https://github.com/mattermost/mattermost-server/blob/master/webapp/channels/src/images/worktemplates/playbooks/feature_lifecycle.png

https://github.com/mattermost/mattermost-server/blob/master/webapp/channels/src/images/worktemplates/product_teams/feature_release/playbooks.png